src/main/java/app/bpartners/geojobs/endpoint/event/gen/DetectionJobCreated.java

```diff
@@ -0,0 +1,19 @@
+package app.bpartners.geojobs.endpoint.event.gen;
+
+import app.bpartners.geojobs.repository.model.geo.tiling.TilingTask;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+import javax.annotation.processing.Generated;
+import lombok.*;
+
+@Generated("EventBridge")
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@EqualsAndHashCode
+@Data
+@ToString
+public class DetectionJobCreated implements Serializable {
```

Comment:
class naming is from action triggering, thus DetectionJobCreated infers that this event gets triggered after a DetectionJob is Created which is incorrect